nacc/uds3/filters_ivp.py

```diff
@@ -0,0 +1,73 @@
+import os
+import sys
+import csv
+import re
+import fileinput
+from nacc.uds3.filters import  write_headers
+
+
+def filter_clean_ptid(input_ptr, filter_meta, output_ptr):
```

Comment:
This function is almost identical to other filter, why not just import that function and only have it written once.